core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3180,12 +3180,12 @@ private void propagatePersistedInternal(Supplier<JournalContext> journalContext,
   }
 
   @Override
-  public void free(AlluxioURI path, FreeContext context)
+  public void free(AlluxioURI path, FreeContext context, JournalContext journalContext)
```

Comment:
maybe i can even change the API to `freeAsyncCommit()` to distinguish from the existing one?